antora-ui-camel/src/partials/footer-content.hbs

```diff
@@ -3,41 +3,86 @@
     </div>
     <footer>
         <div class="footer">
-            <figure class="logo">
-                <img src="{{siteRootPath}}/_/img/logo-d.svg" class="logo-small mt-60" alt="Apache Camel Logo" aria-label="white silhouette of a camel in front of a sand dune">
-            </figure>
             <dl>
+                <figure class="logo">
+                    <img src="{{siteRootPath}}/_/img/logo-d.svg" class="logo-small mt-60" alt="Apache Camel Logo"
+                        aria-label="white silhouette of a camel in front of a sand dune">
+                </figure>
                 <dt>Overview</dt>
-                <dd><a href="{{siteRootPath}}/blog/">Blog</a></dd>
-                <dd><a href="{{siteRootPath}}/components/latest/">Components</a></dd>
-                <dd><a href="{{siteRootPath}}/download/">Download</a></dd>
-                <dd><a href="{{siteRootPath}}/manual/latest/getting-started.html">Getting started</a></dd>
-                <dd><a href="{{siteRootPath}}/manual/latest/faq/index.html">FAQ</a></dd>
+                <input type="checkbox" id="show-overview"/>
+                <label for="show-overview"><img class="show-menu" src="{{siteRootPath}}/_/img/show-footer-menu.svg"></label>
+                <label for="show-overview"><img class="hide-menu" src="{{siteRootPath}}/_/img/hide-footer-menu.svg"></label>
+                <div class="footer-menu">
+                    <dd><a href="{{siteRootPath}}/blog/">Blog</a></dd>
+                    <dd><a href="{{siteRootPath}}/docs/">Documentation</a></dd>
+                    <dd><a href="{{siteRootPath}}/community/support">COmmunity</a></dd>
```

Comment:
@AemieJ this needs to be:
```suggestion
                    <dd><a href="{{siteRootPath}}/community/support/">Community</a></dd>
```

Note the `/` at the end of `support/`, without it the URL would point to a file, with it it points to `index.html` in the `support` directory.